webapp/src/js/actions/global.js

```diff
@@ -177,8 +177,11 @@ angular.module('inboxServices').factory('GlobalActions',
 
       /**
        * Clear the selected item - may update the URL
+       * rename to "back" for android back button and webapp < button?
```

Comment:
is this a TODO but without the TODO at the front? I think you meant to drop it?

Reply:
You're 100% right. Good spotting.